src/firewall/core/ipXtables.py

```diff
@@ -906,7 +906,7 @@ def _rule_addr_fragment(self, opt, address, invert=False):
         elif check_mac(address):
             # outgoing can not be set
             if opt == "-d":
-                raise FirewallError(INVALID_ADDR, "Can't match a destination MAC.")
+                return [""]
```

Comment:
Just an empty list. No need for the empty string.

Reply:
Returning just an empty list causes the test to fail.